### ISSUE TITLE
Navigate through tabs with keyboard

### ISSUE THREAD

**Issue by jeron-diovis** [achekulaev/tabspane#2]
Is there any way to navigate through *opened* tabs with keyboard?

The only thing I currently found is to switch to history pane (under search results) with `alt+tab` – which is not desired, because from there extension opens new tab instead of switching to existing one.

**Comment by achekulaev** [comment]
Right. There's no way to do it at the moment. It's a feature pending

**Comment by jason404** [comment]
I noticed that there's a search engine entry from this expention 'ls'.  

By typing ls, tab, [search entry], <enter>, I would have expect it to open a TabsPane tab with the search entry already input.  Instead, nothing seems to happen.  Does this work in some way?  I have changed the keyboard shortcut, in case that makes any difference.

Has anybody forked this?  It's a great extension, but could be improved.

**Comment by jason404** [comment]
Doesn't matter, I figured out what the search engine entry is for.  It lists all tabs with the phrase entered in the dropdown list and so you can select them using only the keyboard.  Excellent.
